### PR TITLE
release: v1.3.0

### PR DESCRIPTION
**Pull request by claytonparnell**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by balajisankar15 — DISMISSED** [review]

File: test/test_artifacts/v1/autogluon.test.Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM $COSMOS_IMAGE as base
 
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 
-RUN micromamba install --freeze-installed -y conda-forge::pytest
+RUN micromamba install --freeze-installed -y -c conda-forge pytest
```
Comment (claytonparnell) [inline test/test_artifacts/v1/autogluon.test.Dockerfile:6]: In this version, pytest is now installed by default; I found that installing `conda-forge::pytest` leads to error due to existing installation, whereas `-c conda-forge pytest` skips existing installation.

**Comment by balajisankar15** [comment]
Can you attach staleness report @claytonparnell

**Review by balajisankar15 — APPROVED** [review]

**Comment by claytonparnell** [comment]
# Staleness Report: 1.3.0(gpu)                                                                                                                                                  

Package | Current Version in the Distribution image | Latest Relevant Version in Upstream
---|---|---
numpy|1.26.2|1.26.2
jinja2|3.1.2|3.1.2
pandas|2.1.4|2.1.4
altair|5.2.0|5.2.0
${\color{red}boto3}$|1.28.64|1.33.12
ipython|8.18.1|8.18.1
jupyter-lsp|2.2.1|2.2.1
jupyterlab|4.0.9|4.0.9
amazon-codewhisperer-jupyterlab-ext|2.0.1|2.0.1
jupyter-scheduler|2.4.0|2.4.0
amazon-sagemaker-jupyter-scheduler|3.0.5|3.0.5
scipy|1.11.4|1.11.4
scikit-learn|1.3.2|1.3.2
pip|23.3.1|23.3.1
${\color{red}torchvision}$|0.15.2|0.16.1
autogluon|0.8.2|0.8.2
ipywidgets|8.1.1|8.1.1
notebook|7.0.6|7.0.6
aws-glue-sessions|1.0.4|1.0.4
conda|23.11.0|23.11.0
jupyter-ai|2.6.0|2.6.0
jupyter-dash|0.4.2|0.4.2
jupyter-server-proxy|4.1.0|4.1.0
jupyterlab-git|0.50.0|0.50.0
jupyterlab-lsp|5.0.1|5.0.1
${\color{red}keras}$|2.12.0|2.15.0
matplotlib|3.8.2|3.8.2
nodejs|18.18.2|18.18.2
py-xgboost-gpu|1.7.6|1.7.6
thrift_sasl|0.4.3|0.4.3
pyhive|0.7.0|0.7.0
python-gssapi|1.8.3|1.8.3
python-lsp-server|1.9.0|1.9.0
${\color{red}pytorch-gpu}$|2.0.0|2.1.0
sagemaker-headless-execution-driver|0.0.12|0.0.12
sagemaker-jupyterlab-emr-extension|0.1.9|0.1.9
sagemaker-jupyterlab-extension|0.2.0|0.2.0
sagemaker-kernel-wrapper|0.0.2|0.0.2
sagemaker-python-sdk|2.198.0|2.198.0
sagemaker-studio-analytics-extension|0.0.21|0.0.21
sasl|0.3.1|0.3.1
supervisor|4.2.5|4.2.5
${\color{red}tensorflow}$|2.12.1|2.14.0

# Staleness Report: 1.3.0(cpu)

Package | Current Version in the Distribution image | Latest Relevant Version in Upstream                                                                                        [4/1983]
---|---|---
numpy|1.26.2|1.26.2
jinja2|3.1.2|3.1.2
${\color{red}pytorch}$|2.0.0|2.1.0
pandas|2.1.4|2.1.4
altair|5.2.0|5.2.0
${\color{red}boto3}$|1.28.64|1.33.12
ipython|8.18.1|8.18.1
jupyter-lsp|2.2.1|2.2.1
jupyterlab|4.0.9|4.0.9
amazon-codewhisperer-jupyterlab-ext|2.0.1|2.0.1
jupyter-scheduler|2.4.0|2.4.0
amazon-sagemaker-jupyter-scheduler|3.0.5|3.0.5
scipy|1.11.4|1.11.4
scikit-learn|1.3.2|1.3.2
pip|23.3.1|23.3.1
${\color{red}torchvision}$|0.15.2|0.16.1
autogluon|0.8.2|0.8.2
ipywidgets|8.1.1|8.1.1
notebook|7.0.6|7.0.6
aws-glue-sessions|1.0.4|1.0.4
conda|23.11.0|23.11.0
jupyter-ai|2.6.0|2.6.0
jupyter-dash|0.4.2|0.4.2
jupyter-server-proxy|4.1.0|4.1.0
jupyterlab-git|0.50.0|0.50.0
jupyterlab-lsp|5.0.1|5.0.1
${\color{red}keras}$|2.12.0|2.15.0
matplotlib|3.8.2|3.8.2
nodejs|18.18.2|18.18.2
py-xgboost-cpu|1.7.6|1.7.6
thrift_sasl|0.4.3|0.4.3
pyhive|0.7.0|0.7.0
python-gssapi|1.8.3|1.8.3
python-lsp-server|1.9.0|1.9.0
sagemaker-headless-execution-driver|0.0.12|0.0.12
sagemaker-jupyterlab-emr-extension|0.1.9|0.1.9
sagemaker-jupyterlab-extension|0.2.0|0.2.0
sagemaker-kernel-wrapper|0.0.2|0.0.2
sagemaker-python-sdk|2.198.0|2.198.0
sagemaker-studio-analytics-extension|0.0.21|0.0.21
sasl|0.3.1|0.3.1
supervisor|4.2.5|4.2.5
${\color{red}tensorflow}$|2.12.1|2.14.0

**Review by trajanikant — APPROVED** [review]
